tools/transformer/src/atlasTransformations.js

```diff
@@ -67,3 +71,42 @@ module.exports = function runTransformations(openapi) {
 
   return openapi;
 };
+
+// Temporary transformation until new search version is introduced.
```

Comment:
nit: for the next time if there's a Jira issue (or perhaps even a link to another GH issue or PR) it could be great to link so that we remember the whole context